urid/src/feature.rs

```diff
@@ -0,0 +1,124 @@
+//! Thin but safe wrappers for the URID mapping features.
+use core::feature::Feature;
+use core::UriBound;
+use std::ffi::CStr;
+
+pub use sys::LV2_URID as URID;
```

Comment:
Here you publicly assign `URID`s to be `u32`s, however the LV2 URID spec (http://lv2plug.in/doc/html/group__urid.html#afaf7f8fb93fe825b16dcbff8b0d8d4e5) explicitely defines URID to be non-zero, so at the very least a `NonZeroU32` should be used here (and the `map` function should return `Option<URID>`) or else you will probably get UB somewhere.
Also, considering `URIDs` are only defined and created by the host, I think those could actually be a newtype around `NonZeroU32`, where the `map` function would be the only way to create one safely (we can always add an `unsafe` unchecked escape hatch of course, unless you're an host, in which case we could provide a function to do `u32 -> Option<URID>` safely).

Reply:
Yup, you're right about that!